src/vs/workbench/contrib/terminal/browser/media/scrollbar.css

```diff
@@ -3,54 +3,43 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.monaco-workbench .editor-instance .xterm-viewport,
-.monaco-workbench .pane-body.integrated-terminal .xterm-viewport {
+.xterm-viewport {
```

Comment:
Let's keep the `.monaco-workbench` prefix in case vscode in embedded in another app that uses xterm.js